src/core_plugins/region_map/public/region_map_vis_params.html

```diff
@@ -21,6 +21,12 @@
       </div>
     </div>
 
+    <div class="kuiSideBarFormRow" ng-hide="!editorState.params.emsHotLink">
+      <div class="kuiSideBarFormRow__control">
+        Preview <a ng-href="{{editorState.params.emsHotLink}}" target="_">{{editorState.params.selectedLayer.name}} on the Elastic Maps Service</a>
```

Comment:
As we flesh out the UI are we expecting this to be a similar UX to the icon-tooltip as seen in some of the other fields on this page? Or does including a link tag make it too complicated for that?